seqr/views/apis/data_manager_api_tests.py

```diff
@@ -742,40 +770,51 @@ def mock_write(content):
                 self.assertEqual(response.status_code, 200)
                 info = [
                     f'Parsed {params["num_parsed_samples"]} RNA-seq samples',
-                    'Attempted data loading for 1 RNA-seq samples in the following 1 projects: 1kg project nåme with uniçøde',
+                    'Attempted data loading for 2 RNA-seq samples in the following 2 projects: 1kg project nåme with uniçøde, Test Reprocessed Project',
                 ]
                 warnings = ['Skipped loading for the following 1 unmatched samples: NA19675_D3']
                 if params.get('extra_warnings'):
                     warnings = params['extra_warnings'] + warnings
                 file_name = RNA_FILENAME_TEMPLATE.format(data_type)
                 response_json = response.json()
-                self.assertDictEqual(response_json, {'info': info, 'warnings': warnings, 'sampleGuids': [mock.ANY], 'fileName': file_name})
+                self.assertDictEqual(response_json, {'info': info, 'warnings': warnings, 'sampleGuids': [RNA_SAMPLE_GUID, mock.ANY], 'fileName': file_name})
                 deleted_count = params.get('deleted_count', params['initial_model_count'])
-                mock_logger.info.assert_has_calls([mock.call(info_log, self.data_manager_user) for info_log in info])
+                info_log_calls = [mock.call(info_log, self.data_manager_user) for info_log in info]
+                if test_round == 0:
```

Comment:
Would it be neater to use the `data_type` as the condition?

Reply:
no because the model will be created by the first subtest that is run regardless of which data type it is, and the order is not deterministic